src/main/java/nextstep/ladder/LadderGame.java

```diff
@@ -2,20 +2,42 @@
 
 import java.util.List;
 
+import nextstep.ladder.domain.Height;
 import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.LadderBuilder;
+import nextstep.ladder.domain.LadderFrame;
 import nextstep.ladder.domain.Players;
+import nextstep.ladder.domain.ResultOfGame;
+import nextstep.ladder.domain.Results;
 import nextstep.ladder.domain.line.RandomLineStrategy;
 import nextstep.ladder.view.InputView;
 import nextstep.ladder.view.OutputView;
 
 public class LadderGame {
+    public static String ALL_COMMAND = "all";
+    public static String QUIT_COMMAND = "quit";
```

Comment:
상수는 final 키워드를 붙여주시면 좋을 것 같아요.
```suggestion
    public static final String ALL_COMMAND = "all";
    public static final String QUIT_COMMAND = "quit";
```

Reply:
빠트렸네요 ㅠㅠ